07-routing-gossip.md

```diff
@@ -494,12 +494,12 @@ The receiving node:
   - if the specified `chain_hash` value is unknown (meaning it isn't active on
   the specified chain):
     - MUST ignore the channel update.
-  - if `timestamp` is NOT greater than that of the last-received
+  - if `timestamp` is NOT greater than OR equal to that of the last-received
```

Comment:
Maybe replacing `NOT greater than OR equal to` by `lower than` makes it clearer?

Reply:
Thanks for the review! I have applied both your changes to your commitment. I'll also change the timestamp config to 'lower than' rather than 'greater than equal to'